app/uk/gov/hmrc/nisp/models/SPResponseModel.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.nisp.models
 
 import play.api.libs.json.Json
 
-case class SPResponseModel(spSummary: Option[SPSummaryModel], spExclusions: Option[ExclusionsModel] = None)
+case class SPResponseModel(spSummary: Option[SPSummaryModel], spExclusions: Option[ExclusionsModel] = None, niExclusions: Option[ExclusionsModel] = None)
```

Comment:
All arguments are coming as option, but we are not treating them as same way like -> Some(*)  or directly as niExclusions?